src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -493,4 +493,18 @@ public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
     }
 
-}
+    /**
+     * Trims the course ID if it is too long.
+     * @param demoCourseId
+     * @param maximumIdLength length to trim
+     * @return trimmed Course ID
+     */
+    public static String truncateHead(String demoCourseId, final int maximumIdLength) {
+        final int courseIdLength = demoCourseId.length();
+        if (courseIdLength <= maximumIdLength) {
+            return demoCourseId;
+        } else {
+            return demoCourseId.substring(courseIdLength - maximumIdLength);
+        }
+    }
```

Comment:
1. This is supposed to be a utility method. It should not mention CourseID in any way.
2. The comment should mention the truncation happens at the head of string, not tail.